BlueprintUI/Sources/Layout/GridRow.swift

```diff
@@ -0,0 +1,191 @@
+import UIKit
+
+/// Like `Row`, `GridRow` displays a list of items in a linear horizontal layout. Unlike `Row`, `GridRow` provides
+/// convenience for describing columnar layout.
+///
+/// Horizontally, `GridRow` children are stretched to fill the available space. Vertically, children are justified to
+/// fill the space of the tallest child.
+///
+/// Children may be sized proportionally or absolutely. Proportionally-sized children are granted a proportion of
+/// the total layout space **after** absolutely-sized children and margins have been subtracted.
+///
+/// ## Example:
+///
+/// ```
+/// GridRow { row in
+///   row.spacing = 8.0
+///   row.add(child: .proportional(name, 0.75))
+///   row.add(child: .proportional(number, 0.25))
+///   row.add(child: .absolute(status, 50))
+/// }
+/// ```
+///
+/// ## Expected layout:
+///
+/// ┌────────────────────────────┬─┬────────┬─┬──────────────────┐
```

Comment:
Fwiw dropping this into a ``` block will make it render correctly in Xcode's docs preview

Reply:
Oh really? I actually found the existing doesn't render properly because it's not in a block. If I update it properly, I see the following:

![Screen Shot 2021-04-01 at 2 35 09 PM](https://user-images.githubusercontent.com/1542313/113338764-8cb23a00-92f7-11eb-8ad5-4e32b0900f25.png)

Reply:
Ahh, right! I somehow deleted the top ticks. Thanks for the catch!

Reply:
Fixed.